Fixed a bad prompt

LukeWood opened the pull request:
(One last PR, we can't have the first image in the guide being a bad one)

fchollet commented:
I noticed this when editing and changed it to the classic "photograph of an astronaut riding a horse". Let me know if you prefer that one. (only prompt I changed, though I lightly edited some others)

LukeWood commented:
> I noticed this when editing and changed it to the classic "photograph of an astronaut riding a horse". Let me know if you prefer that one. (only prompt I changed, though I lightly edited some others)

photograph of an astronaut riding a horse is a classic; lets roll with that one.
